src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -669,9 +711,11 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
     };
     NativeAdapter._logger.debug('Video element audio track change');
     const vidIndex = getActiveVidAudioTrackIndex();
-    const pkIndex = getActivePKAudioTrackIndex();
+    const activeAudioTrack = this._getActivePKAudioTrack();
```

Comment:
if this can return null/undefined then you have an issue with the types in `_getActivePKAudioTrack`